internal/monitors/kubernetes/cluster/monitor.go

```diff
@@ -7,25 +7,19 @@
 // watch API is used to efficiently maintain the state between read intervals
 // (see `clusterstate.go`).
 //
-// This plugin should only be run at one place in the cluster, or else metrics
```

Comment:
Looks like half the documentation was removed and it's in the actual docs now so I removed the rest.